### PR TITLE
To Do list

### PR DESCRIPTION
**Pull request by felixDev22**
**In this repo**

- [ ] Created a new repository called "To-do-list".
- [ ] Added Github flows and webpack config files.
- [ ] Used HTML webpack plugin to create index.html file.
- [ ] Used Css webpack loaders.
- [ ] Created an array and a function to populate the task on html.
- [ ] checked that it works properly
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+git clone https://github.com/felixDev22/To-do-list
+
+### Run
+
+To run project, run the following command: git clone
+
+Open in desired editor.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (mikethreels) [inline README.md]: - [ ] when following these steps the potential user is not able to run the project, please make the instructions clear on how to run the project locally.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const task = {};
+  task.index = taskArr.length + 1;
+  task.description = addTask.value;
+  task.completed = false;
+  taskArr.push(task);
+};
+const pushToLocal = () => {
+  localStorage.setItem('taskArr', JSON.stringify(taskArr));
+};
+
```
Comment (mikethreels) [inline src/index.js]: - [ ] you are required in this milestone to populate the task array yourself, adding tasks using the form is a requirement for the next milestone. please make sure to follow the instructions.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Mozo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @felixDev22 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 👏🏼 👏🏼 👏🏼
 
## Congratulations! 💯 💯 💯 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excelent work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**